spec/features/admin/tinymce_feature_spec.rb

```diff
@@ -16,13 +16,13 @@
 
   context "with asset host" do
     before do
-      expect(ActionController::Base.config).to receive(:asset_host_set?).and_return(true)
+      ActionController::Base.config.asset_host = "myhost.com"
```

Comment:
You need to reset this in an `after` block to the previous value, otherwise this will stick with the remaining tests

Reply:
Ha, found the culprit :)

Reply:
added a fix